### PR TITLE
Use a reliable image for integration test

### PR DESCRIPTION
**Pull request by tsouza**

### CODE REVIEW

File: metricbeat/module/php_fpm/_meta/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM richarvey/nginx-php-fpm:1.1.2
+FROM tsouza/nginx-php-fpm
```
Comment (ruflin) [inline metricbeat/module/php_fpm/_meta/Dockerfile]: Can you also tag the image? So in case we change it in the future, it will not break older releases.

Comment (tsouza) [inline metricbeat/module/php_fpm/_meta/Dockerfile]: Ok, I'll do it, sorry!

Comment (tsouza) [inline metricbeat/module/php_fpm/_meta/Dockerfile]: I'll tag based on php version. Sounds good?

Comment (ruflin) [inline metricbeat/module/php_fpm/_meta/Dockerfile]: SGTM. Interestingly @richarvey did that also previously and then switched to an other versioning system. Would be interesting to know why.

Comment (tsouza) [inline metricbeat/module/php_fpm/_meta/Dockerfile]: Ok! I'll ping him, maybe he answers

**Comment by ruflin** [comment]
@tsouza Thanks for creating the container and the PR.

**Pull request merged**